docs/plugins/transform-react-inline-elements.md

```diff
@@ -10,14 +10,16 @@ Replaces the `React.createElement` function with one that is more optimized for
 
 <blockquote class="babel-callout babel-callout-info">
   <p>
-    Note that this requires the global polyfill due to React's use of ES6 <code>Symbol</code> when validating React Elements.
-    If <code>Symbol</code> is not present in the browser, your application will fail to render, as Babel will polyfill <code>Symbol</code>
-    but React will not have access to that polyfill.
+    Note that this might require the Symbol's global polyfill due to React's use of ES6 <code>Symbol</code> when validating React Elements.
```

Comment:
i think this
```
Note that this requires Symbol to be available (or polyfilled) globally since React uses them when validating elements.
```
is really misleading, cause this transform doesnt need it at all, it ONLY needs it if:
- we polyfill locally (`'transform-runtime'`)
- OR include external helpers via `transform-runtime` (`['transform-runtime', { helpers: true, polyfill: false }]`), because those helpers come precompiled so they import `Symbol` polyfill even with `polyfill: false`
- and possibly with `external-helpers` too, i guess they might be precompiled too, but didnt use this and hadnt time to check

If we use `transform-react-inline-elements` alone, everything works fine, cause it inlines `_jsx` helper at the top of the module which is not precompiled and has this:
```javascript
var REACT_ELEMENT_TYPE = (typeof Symbol === 'function' && Symbol.for && Symbol.for('react.element')) || 0xeac7;
```
which is not in a conflict with a version within React so they work just ok together